crates/holochain/src/fixt.rs

```diff
@@ -362,7 +349,35 @@ fixturator!(
 
 fixturator!(
```

Comment:
I couldn't figure out how to do this without expanding the definitions... I could wrap the `broadcast::Sender` in a custom type so that it can have a fixturator I guess but I'm not sure whether that's better